From a894b97af8fdc85f770bf579a55b5d317a1f9913 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 24 Dec 2018 17:25:11 +0800
Subject: [PATCH] ARM: dts: rockchip: rk3288-evb-rk808-linux: add cec pinctrl

tested on rk3288-evb-rk808-linux.

The resistance R1036 maybe NC on RK_EVB_RK3288_LPDDR3_V1x
Please check it if you want use the hdmi cec function.

Change-Id: I2533351ea5741b501b25c4c0485dc9246adf11bb
Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-rk808-linux.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts b/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
index f4b7378d5480..2c3099f4c66a 100644
--- a/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
+++ b/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
@@ -235,6 +235,10 @@
 	mali-supply = <&vdd_gpu>;
 };
 
+&hdmi {
+	pinctrl-0 = <&hdmi_ddc>, <&hdmi_cec>;
+};
+
 &i2c0 {
 	clock-frequency = <400000>;
 
-- 
2.35.3

